src/helpers/terrahub.js

```diff
@@ -167,7 +167,7 @@ class Terrahub {
    */
   _getTask() {
     return this._on('start')
-      .then(() => this._hook('before'))
+      .then(() => this._hook('before', {}))
```

Comment:
why before hook accepts default value as `{}` while after hook - `res`?

Reply:
I think this line should look like this: `.then(res => this._hook('before', res))`. Does this make sense?